tutorials/migration.md

```diff
@@ -642,87 +543,73 @@ Back up your openHAB 1.x configurations and uninstall openHAB 1.x if desired.
 
 ## Migrating to openHAB 2 Bindings
 
-### Eclipse SmartHome Designer
+### Integrated Development Environment
 
-As mentioned above, there is a new Integrated Development Environment (IDE) for
-openHAB 2 configurations,
-[Eclipse SmartHome Designer](https://www.eclipse.org/smarthome/documentation/community/downloads.html).
-The old openHAB Designer is not compatible with openHAB 2.
+As mentioned above, openHAB Designer is no longer recommended or supported. There is a new Integrated Development Environment (IDE) for openHAB 2 configurations, [VSCode with the openHAB Extension]{{base}}/configuration/editors.html#openhab-vs-code-extension).
 
-### My.openHAB
+### myopenHAB.org
 
-There is no 1.9 My.openHAB binding that is compatible with openHAB 2, only a
-native 2.0 binding. Furthermore, one can have only one openHAB instance linked to
-a http://my.openhab.org account at a time. This is why we have saved migrating this
-binding until now.
+There is no 1.x My.openHAB binding that is compatible with openHAB 2, only a native 2.0 openHAB Cloud Connector binding. 
+Furthermore, one can have only one openHAB instance linked to a http://myopenhab.org account at a time. 
+This is why we have saved migrating this binding until now.
 
-Each openHAB instance generates its own unique UUID and Secret so one cannot simply
-copy these files from your old openHAB 1.x installation.
+Each openHAB instance generates its own unique UUID and Secret so one cannot simply copy these files from your old openHAB 1.x installation.
 
-Using your preferred method for add-on installation (see above) install the
-My.openHAB binding. Once installed a new `uuid` and `secret` file will be created.
-These two files are located in a different place in openHAB 2:
-`<openhab 2 userdata>/uuid` and `<openHAB 2 userdata>/myopenhab/secret`.
-As of this writing, the http://my.openhab.org documentation still references the
-openHAB 1.x locations.
+Using your preferred method for add-on installation (see above) install the openHAB Cloud Connector binding (Misc tab in PaperUI). 
+Once installed a new `uuid` and `secret` file will be created.
+These two files are located in a different place in openHAB 2: `<openhab 2 userdata>/uuid` and `<openHAB 2 userdata>/myopenhab/secret`.
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.